help/stream-permissions.md

```diff
@@ -99,7 +99,7 @@ subscribed.</span>
 | Change the privacy    | &#9726;           |            |           |
 | Rename                | &#10004;          |            |           |
 | Edit the description  | &#10004;          |            |           |
-| Remove others         | &#10004;          |            |           |
+| Remove others         | &#10004;          | &#10038;   | &#10038;  | &#10038;
 | Delete                | &#10004;          |            |           |
 
 <span class="legend_symbol">&#10004;</span><span class="legend_label">Always</span>
```

Comment:
The link on "Configurable" doesn't quite make a sense anymore (in both places). How about something like this?

> Configurable. See `<page x>` and `<page y>` for details.